spec/features/visitors/sign_up_spec.rb

```diff
@@ -220,19 +220,6 @@
     expect(page).to have_content(invalid_email_message)
   end
 
-  scenario 'visitor signs up with an email already in the system', email: true do
-    user = create(:user, email: 'existing_user@example.com')
-    sign_up_with('existing_user@example.com')
-
-    expect(page).to have_content t('notices.signed_up_but_unconfirmed.first_paragraph_start')
-    expect(page).to have_content t('notices.signed_up_but_unconfirmed.first_paragraph_end')
-    expect(page).
-      to have_content t('notices.signed_up_but_unconfirmed.no_email_sent_explanation_start')
-    expect(page).to have_content user.email
-    expect(page).to have_link(t('links.resend'), href: new_user_confirmation_path)
-    expect(last_email.html_part.body).to have_content 'This email address is already in use.'
```

Comment:
this is being replaced by the spec on L278 (hard to tell bc this was not localized here)